src/metatensor/models/utils/normalize.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Union
+
+import torch
+from metatensor.learn.data.dataset import _BaseDataset
```

Comment:
can we not import a private class? I don't think we need the type annotations here anyway, since this is not required to be torch script compatible

Reply:
Outdated and removed after #83

Reply:
Replaced with `metatensor.learn.data.dataset.Dataset`

Reply:
This is a comment for the rest of the code. I assume @abmazitov just took it from the other code we have

Reply:
I agree with the comment though